src/ServiceControl/Monitoring/KnownEndpointIndex.cs

```diff
@@ -8,15 +8,15 @@ public class KnownEndpointIndex : AbstractIndexCreationTask<KnownEndpoint>
     {
         public KnownEndpointIndex()
         {
-            Map = messages => from message in messages
-                              select new
-                              {
-                                  EndpointDetails_Name = message.EndpointDetails.Name,
-                                  EndpointDetails_Host = message.EndpointDetails.Host,
-                                  message.HostDisplayName,
-                                  message.Monitored,
-                                  message.HasTemporaryId
-                              };
+            Map = knownEndpoints => from knownEndpoint in knownEndpoints
```

Comment:
So to be clear here, you're changing the variable name because this index is on known endpoints (so this index touches a few hundred documents, max) not on processed messages, so it really never should have said `message` in the first place. Correct?

Reply:
Yea